satpy/tests/reader_tests/test_oli_tirs_l1_tif.py

```diff
@@ -0,0 +1,576 @@
+    <RADIANCE_MULT_BAND_10>3.3420E-04</RADIANCE_MULT_BAND_10>
+    <RADIANCE_MULT_BAND_11>3.3420E-04</RADIANCE_MULT_BAND_11>
+    <RADIANCE_ADD_BAND_1>-61.78647</RADIANCE_ADD_BAND_1>
+    <RADIANCE_ADD_BAND_2>-63.27010</RADIANCE_ADD_BAND_2>
+    <RADIANCE_ADD_BAND_3>-58.30286</RADIANCE_ADD_BAND_3>
+    <RADIANCE_ADD_BAND_4>-49.16426</RADIANCE_ADD_BAND_4>
+    <RADIANCE_ADD_BAND_5>-30.08607</RADIANCE_ADD_BAND_5>
+    <RADIANCE_ADD_BAND_6>-7.48213</RADIANCE_ADD_BAND_6>
+    <RADIANCE_ADD_BAND_7>-2.52188</RADIANCE_ADD_BAND_7>
+    <RADIANCE_ADD_BAND_8>-55.64041</RADIANCE_ADD_BAND_8>
+    <RADIANCE_ADD_BAND_9>-11.75832</RADIANCE_ADD_BAND_9>
+    <RADIANCE_ADD_BAND_10>0.10000</RADIANCE_ADD_BAND_10>
+    <RADIANCE_ADD_BAND_11>0.10000</RADIANCE_ADD_BAND_11>
+    <REFLECTANCE_MULT_BAND_1>2.0000E-05</REFLECTANCE_MULT_BAND_1>
+    <REFLECTANCE_MULT_BAND_2>2.0000E-05</REFLECTANCE_MULT_BAND_2>
+    <REFLECTANCE_MULT_BAND_3>2.0000E-05</REFLECTANCE_MULT_BAND_3>
+    <REFLECTANCE_MULT_BAND_4>2.0000E-05</REFLECTANCE_MULT_BAND_4>
+    <REFLECTANCE_MULT_BAND_5>2.0000E-05</REFLECTANCE_MULT_BAND_5>
+    <REFLECTANCE_MULT_BAND_6>2.0000E-05</REFLECTANCE_MULT_BAND_6>
+    <REFLECTANCE_MULT_BAND_7>2.0000E-05</REFLECTANCE_MULT_BAND_7>
+    <REFLECTANCE_MULT_BAND_8>2.0000E-05</REFLECTANCE_MULT_BAND_8>
+    <REFLECTANCE_MULT_BAND_9>2.0000E-05</REFLECTANCE_MULT_BAND_9>
+    <REFLECTANCE_ADD_BAND_1>-0.100000</REFLECTANCE_ADD_BAND_1>
+    <REFLECTANCE_ADD_BAND_2>-0.100000</REFLECTANCE_ADD_BAND_2>
+    <REFLECTANCE_ADD_BAND_3>-0.100000</REFLECTANCE_ADD_BAND_3>
+    <REFLECTANCE_ADD_BAND_4>-0.100000</REFLECTANCE_ADD_BAND_4>
+    <REFLECTANCE_ADD_BAND_5>-0.100000</REFLECTANCE_ADD_BAND_5>
+    <REFLECTANCE_ADD_BAND_6>-0.100000</REFLECTANCE_ADD_BAND_6>
+    <REFLECTANCE_ADD_BAND_7>-0.100000</REFLECTANCE_ADD_BAND_7>
+    <REFLECTANCE_ADD_BAND_8>-0.100000</REFLECTANCE_ADD_BAND_8>
+    <REFLECTANCE_ADD_BAND_9>-0.100000</REFLECTANCE_ADD_BAND_9>
+  </LEVEL1_RADIOMETRIC_RESCALING>
+  <LEVEL1_THERMAL_CONSTANTS>
+    <K1_CONSTANT_BAND_10>774.8853</K1_CONSTANT_BAND_10>
+    <K2_CONSTANT_BAND_10>1321.0789</K2_CONSTANT_BAND_10>
+    <K1_CONSTANT_BAND_11>480.8883</K1_CONSTANT_BAND_11>
+    <K2_CONSTANT_BAND_11>1201.1442</K2_CONSTANT_BAND_11>
+  </LEVEL1_THERMAL_CONSTANTS>
+  <LEVEL1_PROJECTION_PARAMETERS>
+    <MAP_PROJECTION>UTM</MAP_PROJECTION>
+    <DATUM>WGS84</DATUM>
+    <ELLIPSOID>WGS84</ELLIPSOID>
+    <UTM_ZONE>40</UTM_ZONE>
+    <GRID_CELL_SIZE_PANCHROMATIC>15.00</GRID_CELL_SIZE_PANCHROMATIC>
+    <GRID_CELL_SIZE_REFLECTIVE>30.00</GRID_CELL_SIZE_REFLECTIVE>
+    <GRID_CELL_SIZE_THERMAL>30.00</GRID_CELL_SIZE_THERMAL>
+    <ORIENTATION>NORTH_UP</ORIENTATION>
+    <RESAMPLING_OPTION>CUBIC_CONVOLUTION</RESAMPLING_OPTION>
+  </LEVEL1_PROJECTION_PARAMETERS>
+</LANDSAT_METADATA_FILE>
+"""
+
+class TestOLITIRSL1(unittest.TestCase):
+    """Test generic image reader."""
+
+    def setUp(self):
+        """Create temporary images and metadata to test on."""
```

Comment:
We are using pytest mostly and dropping Testcase, so I think you can do this (not tested):
```suggestion
class TestOLITIRSL1:
    """Test generic image reader."""

    def setup_method(self):
        """Create temporary images and metadata to test on."""
```